common/app/model/facia/PressedCollection.scala

```diff
@@ -31,6 +31,26 @@ case class PressedCollection(
   hasMore: Boolean
 ) {
 
+  lazy val adFree = {
+    copy(
+      curated = curated.filterNot(_.isPaidFor),
+      backfill = backfill.filterNot(_.isPaidFor),
+      treats = treats.filterNot(_.isPaidFor)
```

Comment:
what happens if curated/backfill/treats end up empty at this point?

Reply:
Those containers don't appear on testing with facia, although I should explicitly remove them here.